Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2856,6 +2856,21 @@ extension BrowserViewController: ReaderModeBarViewDelegate {
 }
 
 extension BrowserViewController: ContextMenuHelperDelegate {
+    @objc
+    private func image(image: UIImage, didFinishSavingwithError error: NSError?, contextInfo: UnsafeRawPointer?) {
+        
+        if error != nil {
```

Comment:
Remove it and use an early return statement `if error == nil { return }`, code will be less indented this way